docs/environment.yml

```
@@ -1,6 +1,18 @@
 name: xeus-python-kernel
 channels:
-  - https://repo.mamba.pm/emscripten-forge
+  - https://prefix.dev/emscripten-forge-dev
   - conda-forge
 dependencies:
+  - python=3.13
+  - pip
   - xeus-python
+  - requests
+  - jupyter_ydoc=2.1.5
+  - ypywidgets>=0.9.6,<0.10.0
+  - yjs-widgets>=0.3.5,<0.4
+  - comm>=0.1.2,<0.2.0
+  - pydantic>=2,<3
+  - pip:
+    - my-jupyter-shared-drive==0.1.13
+    - python/jupytergis_lab
+    - python/jupytergis_core
```

Since we already built wheels at this point, you should be able to pass

```suggestion
    - python/jupytergis_lab/dist/jupytergis*.whl
    - python/jupytergis_core/dist/jupytergis*.whl
```